string operation_name
float32 number
bool is_multiplication
